platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/EventSignatureValidationWire.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.wiring;
+
+import com.swirlds.common.wiring.InputWire;
+import com.swirlds.common.wiring.OutputWire;
+import com.swirlds.common.wiring.TaskScheduler;
+import com.swirlds.common.wiring.WiringModel;
+import com.swirlds.platform.event.GossipEvent;
+import com.swirlds.platform.event.validation.EventValidator;
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/**
+ * Wiring for the event signature validator.
+ */
+public class EventSignatureValidationWire {
```

Comment:
no longer a "wire"

Reply:
renamed to `EventSignatureValidationScheduler`